src/xc/xc_libxc.F

```diff
@@ -181,8 +181,8 @@ SUBROUTINE libxc_lda_info(libxc_params, reference, shortform, needs, max_deriv)
             CPABORT(TRIM(func_name)//": this XC_FAMILY is currently not supported.")
          END SELECT
       END IF
-      IF (.NOT. (xc_libxc_wrap_is_stable(xc_info))) THEN
-         CPWARN(TRIM(func_name)//" is not considered stable. Use with caution.")
+      IF (.NOT. (xc_libxc_wrap_is_under_development(xc_info))) THEN
```

Comment:
shouldn't `xc_libxc_wrap_is_under_development` return `.TRUE.` if it is under development?